header/p2p/exchange.go

```diff
@@ -130,22 +139,34 @@ func (ex *Exchange) performRequest(
 	if err != nil {
 		return nil, err
 	}
+	if err = stream.SetWriteDeadline(time.Now().Add(writeDeadline)); err != nil {
+		log.Warnf("error setting deadline: %s", err)
```

Comment:
I would demote all warn logs to debug level here since we don't actually care *too* much about this

Reply:
ok

Reply:
done.

Reply:
Note that libp2p folks use Warn in this case.